internal/cli/server/flags.go

```diff
@@ -167,6 +169,12 @@ func (c *Command) Flags(config *Config) *flagset.Flagset {
 		Value:   &c.cliConfig.Heimdall.URL,
 		Default: c.cliConfig.Heimdall.URL,
 	})
+	f.DurationFlag(&flagset.DurationFlag{
+		Name:    "bor.heimdalltimeout",
+		Usage:   "Timeout of Heimdall service",
+		Value:   &c.cliConfig.Heimdall.Timeout,
+		Default: 5 * time.Second,
```

Comment:
Actually what you did earlier was correct. 

You can add the default value here:
https://github.com/maticnetwork/bor/blob/3154fc4e0e1a1db56f95eac08a8929a3552940be/internal/cli/server/config.go#L647

Now you can use the variable directly like done in other flags.

Reply:
With the default being set in `config.go`, you won't need a hardcoded value anymore. You can directly use `c.cliConfig.Heimdall.Timeout` here in `Default` field.